scale-tests/setup_streaming.py

```diff
@@ -75,6 +76,11 @@ def install_package(package_name: str,
 
     service_options = scale_tests_utils.get_service_options(service_name, service_account_info, additional_options, config_path)
 
+    if sdk_marathon.app_exists(service_name):
+        log.info("%s index %s as %s is already installed", package_name, index, service_name)
+        log.info("Uninstalling %s index %s as %sd", package_name, index, service_name)
+        sdk_install.uninstall(package_name, service_name)
```

Comment:
Ideally we wouldn't uninstall already running services, but instead just make sure that their information are in the `infrastructure.json` file.